cmake/INSTALL_USING_CMAKE

```diff
@@ -146,7 +146,7 @@ variables offered to the user in the CMake console and GUI (ccmake and
 cmake-gui, respectively). Each variable is annotated with a brief
 documentation string. The most important variables are:
 
-BUILD_SHARED_LIBS             Build shared instead of static libraries
+BUILD_SYSTEMC_SHARED_LIBS     Build shared instead of static libraries
```

Comment:
**Attention:** Resolve inconsistency: `BUILD_SYSTEMC_SHARED_LIBS` -> `SYSTEMC_BUILD_SHARED_LIBS`

Reply:
Fixed.
Another issue is that not all cmake options are listed here.

Reply:
Thanks! It would be great if you could add the ones you are aware of or create a corresponding issue listing them.